Assets/Scripts/SelectionSquare.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class SelectionSquare : MonoBehaviour, IPointerClickHandler
+{
+    public GameObject my_placement_object;
```

Comment:
We need to establish proper styles before we get too far into this. `camelCase` is the C# standard.

Reply:
fixed